### PR TITLE
Fixes #24748 - Add Cluster option to Hammer CR.

### PR DESCRIPTION
**Pull request by chris1984**

### CODE REVIEW

**Comment by chris1984** [comment]
@ares can I get a review please?

**Comment by chris1984** [comment]
Output of change is below:
```
hammer> compute-resource clusters --id 1
----------------------|-----------
NAME                  | ID        
----------------------|-----------
Satellite_Engineering | domain-c7 
SysMgmt_vMotion       | domain-c20
----------------------|-----------
```

File: lib/hammer_cli_foreman/compute_resource.rb [hunk 1]
```diff
@@ -108,6 +108,30 @@ class DeleteCommand < HammerCLIForeman::DeleteCommand
...
+     def request_params
+       params = super
+       params['id'] ||= get_identifier
+       params
+     end
+
+     output do
+       field :name, _('Name')
+       field :id,   _('Id')
+     end
```
Comment (akofink) [inline lib/hammer_cli_foreman/compute_resource.rb]: This is indented only 1 space instead of 2

**Comment by chris1984** [comment]
@akofink updated with a test, when I look at the test file in Vim or Atom it shows the ends in the correct spot in the test file so not sure why Github is showing that.

File: lib/hammer_cli_foreman/compute_resource.rb (hunk 1 above)
Comment (chris1984) [reply lib/hammer_cli_foreman/compute_resource.rb]: Looks fine in Vim/Atom

**Comment by chris1984** [comment]
@akofink tested just to make sure nothing was wrong with my cloned repo on the fs, i recloned my repo to /tmp and did checkout -b blah --track origin/add-cluster and still see the ends are in the right spot.

File: lib/hammer_cli_foreman/compute_resource.rb (hunk 1 above)
Comment (chris1984) [reply lib/hammer_cli_foreman/compute_resource.rb]: NM see what you mean and fixed,.

**Comment by chris1984** [comment]
@akofink updated!

**Comment by ares** [comment]
I'll give it a quick test when I have a moment, but LGTM (also don't have perms here)

**Comment by chris1984** [comment]
@mbacovsky Can I get a merge on this?

**Comment by tstrachota** [comment]
:+1: thank you @chris1984, this is a useful command we've been missing for a long time.

**Pull request merged**